salt/roster/dir.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+'''
+Read in the rosters in a directory using the renderer system.
```

Comment:
Please provide some documentation on how to set this up and how it works.

Thanks,
Daniel